src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Circle implements Figure {
```

Comment:
Class names should not be verbs. In this case, 'Circle' is appropriate, so there is no issue with the class name.